fastlane/Pluginfile

```diff
@@ -9,5 +9,3 @@ end
 gem 'fastlane-plugin-wpmreleasetoolkit', '~> 2.0'
 gem 'fastlane-plugin-sentry', '~> 1.0'
 gem 'fastlane-plugin-appcenter', '~> 1.6'
```

Comment:
I worked on this PR to [close the loop on the aborted upgrade in #6114](https://github.com/woocommerce/woocommerce-ios/pull/6114#issuecomment-1034373082) because I already played a similar script [in WordPress iOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/17456/).

In the original PR on this repo, there's an upgrade to version 2.0 of the App Center plugin. I didn't do it in this one because I figured a major version bump was something best left in its own PR. Also, I thought this would take me only a couple of minutes but I had to tweak the work a few times already 🙃